com.dynamo.cr/com.dynamo.cr.bob/scripts/copy.sh

```diff
@@ -84,10 +84,10 @@ cp -v $DYNAMO_HOME/ext/lib/x86_64-win32/wrap_oal.dll lib/x86_64-win32/wrap_oal.d
 
 rm -rf tmp
 mkdir -p tmp
-tar xf ../../packages/luajit-2.1.0-633f265-win32.tar.gz -C tmp
-tar xf ../../packages/luajit-2.1.0-633f265-x86_64-win32.tar.gz -C tmp
-tar xf ../../packages/luajit-2.1.0-633f265-x86_64-linux.tar.gz -C tmp
-tar xf ../../packages/luajit-2.1.0-633f265-x86_64-macos.tar.gz -C tmp
+tar xf ../../packages/luajit-2.1.0-6c4826f-win32.tar.gz -C tmp
+tar xf ../../packages/luajit-2.1.0-6c4826f-x86_64-win32.tar.gz -C tmp
+tar xf ../../packages/luajit-2.1.0-6c4826f-x86_64-linux.tar.gz -C tmp
+tar xf ../../packages/luajit-2.1.0-6c4826f-x86_64-macos.tar.gz -C tmp
```

Comment:
I'm still not a fan of copy.sh. Once again, I forgot it :shrug:

Reply:
Can we automate it somehow? Make it a part of a build step?

Reply:
I should think so, but I haven't given it much attention (yet)